util/util.go

```diff
@@ -148,3 +148,21 @@ func RemoveOrphanedInstances(verbose bool) error {
 
 	return nil
 }
+
+func DeepCopyMap(original map[string]string) map[string]string {
+	res := make(map[string]string, 15)
```

Comment:
Once more (I am just referencing all in case you decide to remove/change something).